drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -36,6 +36,15 @@
 #define z_mco2_prescaler(v) LL_RCC_MCO2_DIV_ ## v
 #define mco2_prescaler(v) z_mco2_prescaler(v)
 
+/* Calculate MSI freq for the given range(at RUN range, not after standby) */
+#if defined(CONFIG_SOC_SERIES_STM32WBX)
+#define Z_RCC_CALC_MSI_RUN_FREQ(range) __LL_RCC_CALC_MSI_FREQ( \
```

Comment:
Z_ namespace is reserved for other purposes (previous `z_` macros should be fixed).

Reply:
is now called RCC_CALC_MSI_RUN_FREQ instead.